### DISCUSSION TITLE
762

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#929] [Comments]
# 762

최근 술자리에서 "왜 혹독한 조언 혹은 높은 프레셔 등 개발팀원들의 성장에 대한 강한 피드백을 하지 않느냐" 에 대한 질문을 들었다. 그에 대해 다음과 같이 답변을 드렸다. “여러 회사를 거치면서 어떤 것이 진짜 성장인지 명확하지 않다는 생각을 하게 되었고, 지금 우리 팀에 필요한 것은 대기업에서 사용하는 기술을 적용하느냐 마느냐가 아니다. 특히 어떠한 한명의 슈퍼맨에 의지해서 조직의 기술력이 높아지는 것 역시 필요하지 않다. 그렇게 해도 팀 전체는 전혀 성장하지 못한 불안한 성장이기 때문이다. 가능하면 다 같이 성장해서 모두가 조직의 졸업까지 이탈자 없는 성장을 바라고 있다.” 만약 현재 우리 팀원들이 제대로 된 성과를 내지 못하고 있다면, 당연히 피드백을 했을 것이고, 잘못된 방향으로의 성장에 대해서..

https://jojoldu.tistory.com/762

**Comment by PilhwanKim** [comment]
돌아보면 저도 비슷한 경험인거 같아요.
저희 어머니는 사교성 있게 살으라고 압박 조언을 주로 했었는데 오히려 사춘기인 저의 반감만 샀었죠.
실제로 사교성을 기른건 저를 있는 그대로 인정해준 집단에 속하고 나서부터였습니다. 
자연스레 그 사람들과 친해진 다음에 사람과의 사귐을 배워나간거죠. 
생각해보면 개인적인 관심과 존중이 개인을 변화시키는 키 포인트인거 같습니다.

**Comment by Chadolbaegi128** [comment]
압박성 충고를 하는 자신에 심취하는 것이 위험한 부분이군요

**Comment by M110-SASS** [comment]
압박하고 공포감을 조성하는 사람치고 제대로 일을 끌어가는 걸 본 적이 없습니다.
절벽으로 몰아간다는 헤드라인 텍스트가 참 와닿네요.

나르시스트라고도 불리는 사람들이 남을 몰아세우는 한편 관리/감독을 한다며 일에 대해 이것저것 간섭하곤 하는데
정작 비즈니스, 조직의 목표에 대해 공부하지도 않고 자신이 말한 것도 기억하지 못하거나 수시로 뒤엎고
그때 그때 느낌과 왜곡된 기억, 감정과 의식의 흐름에 따라 이래라 저래라 하기 일쑤고

귀에 거슬리거나 신경쓰이는 걸 보면 매번 "뭐 했냐"며 소리 지르는 걸 여럿 봤습니다. 그런 회사들은 하나 같이 평점이 1 ~ 2 사이를 맴돌더군요.

하나도 도움이 안되는 사람들이었습니다.
오히려 그들 또는 개인의 이익을 위한 목표와 목적은 달성해가는 반면
소속된 팀원들과 조직 그리고 회사도 점점 병들어가며 아무도 책임지지 않고 아무도 생각하지 않는 투자금만 까먹는 회사가 되곤 했었죠.

혹독한 직언은 폭언이 되기 쉽고
리더가 책임을 다하는 대신 니가 더 잘하라며 가하는 그런 압박감은 정직하게 노력하는 사람을 농락하고 그들의 노력을 악용하는 것으로 변질되기 쉽습니다.

본문이 좋은 말을 써놨다고 해서 공감하는 게 아닙니다.
막상 저런 일상을 겪어 보면, 피드백? 조언? 충고? 아니요. 사람이 일터에서 뛰어 내립니다.
불행히도 저는 그런 사람들과 일을 해보고 정말 이 사회, 우리 업계가 이상한 인간들이 적지 않구나 부조리가 많구나
인간은 이렇게 밖에 살 수 없나 싶은 회의감을 느끼곤 했었습니다.

좋은 리더가 있을 땐 좋은 리더를 모르고 넌 왜 물렀냐, 왜 뼈아프게 충고하지 않냐는 둥의 말들은 죄송하지만 참 속 편한 소리 같에요.

결국 좋은 리더가 되려도 부단히 노력 중인 눈 앞에 타인을 탓하기 바쁜 거죠.
회초리를 들지 않아도 성장하고 공부해서 두각을 나타내는 인물들은 하나 같이 선생님들을 탓하지 않아요.

부끄럽지만 개발을..대충하고 살아서 별 볼일 없는 사람이지만 혹독한 조언과 높은 프래셔는 
남이 나한테 가하는 게 결코 좋은 게 아니며 그건 자기 자신이 자신을 다스리고 가꾸는데 스스로 매질하는데나 필요한 것이죠.
남한테 바랄 게 아닙니다.
